org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/imports/MultiImportOrganizer.java

```diff
@@ -75,7 +75,7 @@
 					}
 					TextEdit textEdit = replaceConverter.convertToTextEdit(replace);
```

Comment:
Why does it produce a null value?

Reply:
If replace.isEmpty then textEdit is null

Reply:
Don't think it matters a lot

Reply:
Adapted as suggested. Makes sense.